packages/sdk/src/types.ts

```diff
@@ -1124,6 +1147,8 @@ interface AccountInfoCommon {
      * This was introduced with node version 7.0
      */
     accountAvailableBalance: CcdAmount.Type;
+    /** The protocol leveltokens held by the account. */
+    accountTokens: TokenAccountInfo[];
```

Comment:
In the protobuf types this is defined as a list. We could be more precise and use a `Map<TokenId.Type, TokenAccountState>` here. Might be harder to use on the consumer end though without providing much value. athis also assumes that we can rely on the uniqueness of the token ID. being enforced. I think for the sake of simplicity, it's better to keep it as is.

Reply:
I agree, protobuf have a map type, but this is also just a wrapper around a list and does not rely on any properties of the key, meaning no performance gains, so the user might as well build the map when that is the most useful to work with.